README.md

```diff
@@ -88,6 +88,9 @@ version = "*"
 default-features = false
 features = ["runtime-agnostic"]
 ```
+## Ecosystem
+
+- [tower-lsp-boilerplate](https://github.com/IWANABETHATGUY/tower-lsp-boilerplate) a github template that let you write a language server with `tower-lsp` more easily.
```

Comment:
Would be nice if we could also fix the capitalization and maybe tweak the wording a little?

```suggestion
- [tower-lsp-boilerplate](https://github.com/IWANABETHATGUY/tower-lsp-boilerplate) - Useful GitHub project template which makes writing new language servers easier.
```